src/style.css

```diff
@@ -1,3 +1,69 @@
+:root {
+  --primary-color: #00acdc;
+  --font-family: font-family: Roboto, Arial, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+a,
+h1,
+h2,
+h3,
+p,
+ul {
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+}
+
+.header {
+  margin: 0 50px;
+  padding: 10px 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav-list {
+  display: flex;
+  list-style: none;
+}
+
+.nav_link {
+  font-family: var(--font-family);
+  font-size: 12px;
+  text-transform: uppercase;
+  color: #000;
+  font-weight: 500;
+  font-style: normal;
+}
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. For example: `font-family: var(--font-family), Arial, sans-serif;`